assets/js/wfmodule/WfModuleDragDropConfig.js

```diff
@@ -0,0 +1,102 @@
+import { DropTarget, DragSource } from 'react-dnd'
+import flow from 'lodash.flow'
+import { findDOMNode } from 'react-dom'
+import {store, reorderWfModulesAction, insertPlaceholderAction, reorderPlaceholderAction} from "../workflow-reducer";
+import debounce from 'lodash/debounce'
+
+export const targetSpec = {
```

Comment:
Can you comment what the purpose of this class is and some high level notes on the logic here?
Also, any way to test this code?